app/features-json/user_json_controller.rb

```diff
@@ -0,0 +1,58 @@
+require_relative "../services/user_service"
+require_relative "api_controller"
+require_relative "json_params"
+
+module FastlaneCI
+  # Controller responsible for handling users
+  class UserJSONController < APIController
+    disable(:authentication)
+
+    HOME = "/api/user"
+
+    post HOME.to_s do
+      email = params[:email]
```

Comment:
I was wondering about this. Should we allow email as a param? Someone could contact our endpoint directly and set a random email. But pulling it from the token makes sure that it is always 1:1 with github or other providers.

Reply:
Ah, I assume it should be possible, but you're right, let's remove it for now until we need it